database/migrations/2020_09_12_225300_create_documents_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateDocumentsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('documents', function (Blueprint $table) {
+            $table->id();
+
+            $table->foreignId('item_id')->constrained();
+
+            $table->string('name', 256);
+            $table->string('path', 256);
+            $table->string('mime_type', 32);
```

Comment:
Esse valor precisa ser [no mínimo 127](https://stackoverflow.com/questions/643690/maximum-mimetype-length-when-storing-type-in-db#:~:text=1%20Answer&text=According%20to%20RFC%204288%20%22Media,can%20be%20max%20127%20characters.) Vamos usar `128` e não teremos problemas.